arviz/tests/base_tests/test_plots_matplotlib.py

```diff
@@ -1185,6 +1185,7 @@ def test_plot_mcse_no_divergences(models):
             "prior_kwargs": {"plot_kwargs": {"color": "r"}},
             "posterior_kwargs": {"plot_kwargs": {"color": "b"}},
         },
+        {"var_names": ["theta"], "kind": "latent"},
```

Comment:
latent is the default, better to use `"observed"` to make sure code does not depend on number of groups and check so every time the code is modified